presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -305,13 +305,22 @@ Property Name                                        Description
                                                      running in EC2, or when the catalog is in a different region.
                                                      Example: ``us-east-1``
 
+``hive.metastore.glue.endpoint-url``                 AWS Glue API endpoint URL.
```

Comment:
This is optional I think..

Reply:
True. This is completely optional. I'll edit accordingly.